NasdaqDataLink.cs

```diff
@@ -57,13 +59,26 @@ static NasdaqDataLink()
             // The NasdaqDataLink API now requires TLS 1.2 for API requests (since 9/18/2018).
             // NET 4.5.2 and below does not enable this more secure protocol by default, so we add it in here
             ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
+
+            var potentialNasdaqToken = Config.Get("nasdaq-auth-token");
+            var potentialQuandlToken = Config.Get("quandl-auth-token");
+
+            if (potentialNasdaqToken.Length != 0)
+            {
+                SetAuthCode(potentialNasdaqToken);
+            } 
+            else if (potentialQuandlToken.Length != 0)
+            {
+                SetAuthCode(potentialQuandlToken);
+                Log.Error("Quandl is obsolete. Use NasdaqDataLink instead.");
```

Comment:
Log error should follow logging style -> `"NasdaqDataLink(): 'quandl-auth-token' is obsolete please use 'nasdaq-auth-token' instead."`